src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/ChangeOfPartyRequest.cs

```diff
@@ -100,7 +100,7 @@ private ChangeOfPartyRequestType GetValidRequestTypeForOriginator(Party originat
             }
         }
 
-        private void CheckPrice(int price)
+        private void CheckPrice(int? price)
```

Comment:
this check will fail for a null price

Reply:
agreed, thought of doing precise level in COP story. done

Reply:
could we do check that if WithParty = Provider then the validation is true otherwise not? Because we don't want a provider to be able to submit null

Reply:
done